test/BUILD

```diff
@@ -149,6 +148,9 @@ codechecker_test(
     tags = [
         "manual",
     ],
+    analyze = [
+        "--analyzers clangsa clang-tidy",
```

Comment:
Should not be here

Reply:
Are you sure? On machines where gcc static analyzer and infer are also enabled, or another one later down the line, the test fails, hence the limit.

Reply:
We should support gcc in the rules in this case or disable it by default.

Reply:
Very well. My thinking was that this is a test of the new bazel rules, not CodeChecker, so I figured its fine to have a test which is a little less fragile with different CodeChecker versions.

Reply:
One more thing; we don't need to change the rules at all for each supported analyzer, correct?